HousingRepairsOnlineApi.Tests/UseCasesTests/SaveRepairRequestUseCaseTests.cs

```diff
@@ -11,16 +11,16 @@ public class SaveRepairRequestUseCaseTests
     {
         private readonly SaveRepairRequestUseCase systemUnderTest;
         private readonly Mock<ISoREngine> mockSorEngine;
-        private readonly Mock<IRepairStorageGateway> mockCosmosGateway;
+        private readonly Mock<IRepairStorageGateway> mockDynamoDBGateway;
```

Comment:
Please use a generic name as these tests don't depend on a specific implementation.
```suggestion
        private readonly Mock<IRepairStorageGateway> mockRepariStorageGateway;
```